doc/man1/req.pod

```diff
@@ -242,7 +242,6 @@ days to certify the certificate for. The default is 30 days.
 
 serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
-It is possible to use negative serial numbers but this is not recommended.
```

Comment:
It used to be possible, then 1.1.0 it wasn't.  And now, actually, it is.  However, a negative serial number is against all the standards, not to mention against all sense of morality and decency :)  I'd rather just not document the possibility.  I'd also like to add a check for ```BN_is_negative()``` but that might break some people who use this for testing.   Thoughts?